### PR TITLE
Create NEW package for mariner 3.0: libtpms

### PR DESCRIPTION
**Pull request by bfjelds**
NEW PACKAGE

###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [ ] The toolchain has been rebuilt successfully (or no changes were made to it)
- [ ] The toolchain/worker package manifests are up-to-date
- [ ] Any updated packages successfully build (or no packages were changed)
- [ ] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [ ] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [ ] All package sources are available
- [ ] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [ ] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [ ] All source files have up-to-date hashes in the `*.signatures.json` files
- [ ] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [ ] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 
Import libtpms (dependency of swtpm, which is dependency of libvirt 10.0.0)

###### Change Log  
- Import from fedora

###### Does this affect the toolchain?  
**NO**

###### Test Methodology
- https://dev.azure.com/mariner-org/mariner/_build/results?buildId=488399&view=results

### CODE REVIEW

File: SPECS/libtpms/libtpms.spec [hunk 1]
```diff
@@ -0,0 +1,345 @@
+Name:           libtpms
```
Comment (mfrw) [inline SPECS/libtpms/libtpms.spec:1]: I think we have missed to add an entry to licenses.

Typically something like this helps:
```bash
./toolkit/scripts/license_map.py --no_check --update --remove_missing \
        SPECS/LICENSES-AND-NOTICES/data/licenses.json \
        SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md \
        SPECS SPECS-EXTENDED SPECS-SIGNED
```

Comment (bfjelds) [reply SPECS/libtpms/libtpms.spec:1]: the command you pasted didn't change any files in my repo.  what should it be doing?

Comment (bfjelds) [reply SPECS/libtpms/libtpms.spec:1]: maybe this isn't recognized by our tooling?

`License:        BSD and TCGL`

Comment (mfrw) [reply SPECS/libtpms/libtpms.spec:1]: A representative PR:
- https://github.com/microsoft/CBL-Mariner/pull/6063/files#diff-74e8b3639b8ff8af22317c85e1868d1bd81f0f6bd24011a64af4058417c370eeR670

A new package has two additional entries:
- licenses.json
- LICENSE-MAP.md

Comment (bfjelds) [reply SPECS/libtpms/libtpms.spec:1]: thanks!  is it as simple as: if the spec is imported from fedora, the package goes into the fedora section?

Comment (mfrw) [reply SPECS/libtpms/libtpms.spec:1]: Yes!

File: SPECS/libtpms/libtpms.spec [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+- Changes following Fedora review comments
+
+* Thu Dec 2 2010 Stefan Berger
+- Small tweaks after reading the FedoreCore packaging requirements
+
+* Tue Nov 16 2010 Stefan Berger
+- Created initial version of rpm spec files
+- Version of library is now 0.5.0
+- Debuginfo rpm is built but empty -- seems to be a known problem
+  Check https://bugzilla.redhat.com/show_bug.cgi?id=209316
```
Comment (ddstreetmicrosoft) [inline SPECS/libtpms/libtpms.spec:346]: missing EOF LF

File: SPECS/libtpms/libtpms.signatures.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+    "Signatures": {
+      "gpgkey-B818B9CADF9089C2D5CEC66B75AD65802A0B4211.asc": "27e7c4bfe417e19582488c9d407676821f39d9538b89e8cc06880a01df5d7e46",
+      "libtpms-0.9.6.tar.gz": "2807466f1563ebe45fdd12dd26e501e8a0c4fbb99c7c428fbb508789efd221c0",
+      "libtpms-0.9.6.tar.gz.asc": "5e9a847a74104d3147b12194e15056a97621faeb34e48aba8ea5955c35d039cc"
+    }
+  }
```
Comment (ddstreetmicrosoft) [inline SPECS/libtpms/libtpms.signatures.json:7]: missing EOF LF

File: SPECS/libtpms/libtpms.spec [hunk 4]
```diff
@@ -0,0 +1,347 @@
+Name:           libtpms
+Version:        0.9.6
+Release:        4%{?dist}
```
Comment (bfjelds) [inline SPECS/libtpms/libtpms.spec:3]: should version=1?  this is the first version for mariner ... 4 comes from photon.

Comment (bfjelds) [reply SPECS/libtpms/libtpms.spec:3]: maybe it should be 5?

Comment (bfjelds) [reply SPECS/libtpms/libtpms.spec]: 5 is the preference.

**Pull request merged**